plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/ProgressMonitor.java

```diff
@@ -19,7 +20,7 @@
  * @author Alexander Garagatyi
  */
 public interface ProgressMonitor {
-    void updateProgress(ProgressStatus currentProgressStatus);
```

Comment:
Throwing this exception here doesn't make sense because this class doesn't declare any dependency on line consumer